src/Range.tsx

```diff
@@ -66,6 +66,12 @@ class Range extends React.Component<IProps> {
       checkBoundaries(value, this.props.min, this.props.max)
     );
     translateThumbs(this.getThumbs(), this.getOffsets(), this.props.rtl);
+
+    values.forEach(value => {
+      if (((value - min) % step)) {
+        console.warn('The `values` property is in conflict with the current `step`, `min` and `max` properties. Please provide values that other accessible using the min, max an step values');
```

Comment:
```suggestion
        console.warn('The inital `values` property is in conflict with the current `step`, `min` and `max` properties. Please provide values that other accessible using the min, max an step values');
```

Reply:
`values that other accessible` should be `values that are accessible`